pkg/workflows/ring/pb/consensus.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package ring;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/workflows/ring/pb";
+
+message Observation {
+  map<uint32, bool> shardHealthStatus = 1; // shard_id -> is_healthy
+  repeated string workflowIDs = 2;
+  google.protobuf.Timestamp now = 3;
+}
+
+message WorkflowRoute {
+  uint32 shard = 1;
+  google.protobuf.Timestamp expires_at = 2;
```

Comment:
why does this have an expiry? If there's no scale event, there's no reason to publish a new route. Also, it's better to fail open than to fail closed (i.e. try the latest thing you have instead of dropping the traffic).